cookbooks/aws-parallelcluster-environment/test/controls/cfnconfig_mixed_spec.rb

```diff
@@ -9,17 +9,19 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'cfnconfig_file_configuration' do
+control 'tag:install_cfnconfig_file_configuration' do
+  on_docker = os_properties.on_docker?
   title 'Check the creation of cfnconfig file'
 
   describe file('/etc/parallelcluster/cfnconfig') do
     it { should exist }
     its('content') { should match /^cfn_region=\w+/ }
-    its('content') { should match /^cfn_scheduler_slots=\w+/ }
+    its('content') { should match /^cfn_scheduler_slots=\w+/ } #TOFIX: this fails
```

Comment:
suggestion: it's better to use TODO or FIXME keywords that are recognized by IDE

Reply:
Will do!